### PR TITLE
Added 3 new integration tests.

### PR DESCRIPTION
**Pull request by jrbenny35**
1. ```should be able to delete a note```: tests that a user is able to delete a note.
2. ```should be able to add a note from the note page```: tests that you can create a new note from the note page.
3. ```should be navigate to feedback page```: tests that the 'Give Feedback' button routes to the correct page.

### CODE REVIEW

File: test/integration/testFirefoxWebext.js [hunk 1]
```diff
@@ -91,4 +91,39 @@ describe('The Firefox Notes web extension', function() {
...
+  });
+
+  it('should be able to add a note from the note page', async function() {
+    // Add a note from the Note Page
+    let listPage = await new ListPage(driver).waitForPageToLoad();
+    let notePage = await listPage.newNoteButton();
+    let title = "THIS IS A TEST";
+    let paragraph = "this isnt a test";
+    await notePage.addNote(title, paragraph)
+    expect(await notePage.noteTitle).to.equal(paragraph)
```
Comment (pdehaan) [inline test/integration/testFirefoxWebext.js]: A few more missing semicolons here, but maybe we're not running ESLint against the test directories.

This also had me a bit confused as it looks like we're asserting that the `noteTitle` is equal to the `paragraph` (I kind of expected it to equal the `title`, but I don't know the underlying data/objects).

**Comment by pdehaan** [comment]
Overall looks great. I had a bit of confusion around why ESLint isn't enforcing rules in these files, but that's outside the scope of this PR.

Also confused why we're adding an updated package-lock.json file when the dependencies don't seem to have changed.

R+=me

File: test/integration/page_objects/note_page.js [hunk 2]
```diff
@@ -42,14 +49,52 @@ export default class NotePage extends BasePage {
...
+
   /**
   * @function addNote
   * @param  {string} title Title of the note you want top add
   * @param  {string} paragraph Paragraph of the note you want to add
   * @returns {Object} An object representing the page.
   * @throws ElementNotFound
   */
-  async addNote(title, paragraph) {
+  async addNote(title = "This is a title", paragraph = 'This is a test!') {
```
[the remark this answers is not in the document]
Comment (jrbenny35) [reply test/integration/page_objects/note_page.js]: I'll fix these.

File: test/integration/testFirefoxWebext.js [hunk 3]
```diff
@@ -91,4 +91,39 @@ describe('The Firefox Notes web extension', function() {
     expect(listNoteTitle).to.equal(paragraph);
   });
 
+  it('should be able to delete a note', async function() {
+    // Deletes a note
+    let listPage = await new ListPage(driver).waitForPageToLoad();
+    let notePage = await listPage.newNoteButton();
+    let title = "THIS IS A TEST";
+    await notePage.addNote(title)
```
[the remark this answers is not in the document]
Comment (jrbenny35) [reply test/integration/testFirefoxWebext.js]: So I figured that is what the original delete test was doing so I didn't need to do that again.

File: test/integration/testFirefoxWebext.js (hunk 1 above)
Comment (jrbenny35) [reply test/integration/testFirefoxWebext.js]: I think I need to change the nomenclature. The title and paragraph doesn't make much sense.

Comment (jrbenny35) [reply test/integration/testFirefoxWebext.js]: @cedricium that helps a bunch actually. Thanks!

Comment (jrbenny35) [reply test/integration/testFirefoxWebext.js]: Looks like I just had the 'enter' command in the wrong place.

**Comment by vladikoff** [comment]
Thanks!

**Pull request merged**